README.md

```diff
@@ -23,6 +23,23 @@ conda install -c conda-forge sleepecg
 The [contributing guide](https://github.com/cbrnr/sleepecg/blob/main/CONTRIBUTING.md) contains detailed instructions on how to contribute to SleepECG.
 
 
+## Dataset reading
+The goal is to have a consistent functional interface for downloading and reading all common polysomnography datasets. While reader functions are a [WIP](https://github.com/cbrnr/sleepecg/pull/28), SleepECG already provides a Python interface for downloading datasets from the _National Sleep Research Resource_ (NSRR) on [sleepdata.org](https://sleepdata.org/), which replicates the functionality of the [NSRR Ruby Gem](https://github.com/nsrr/nsrr-gem).
```

Comment:
```suggestion
SleepECG provides a consistent functional interface for downloading and reading common polysomnography datasets. While reader functions are a [WIP](https://github.com/cbrnr/sleepecg/pull/28), SleepECG already provides a reader for datasets from the _National Sleep Research Resource_ (NSRR) on [sleepdata.org](https://sleepdata.org/), which replicates the functionality of the [NSRR Ruby Gem](https://github.com/nsrr/nsrr-gem).
```

Reply:
The module `nsrr` just downloads data, so I'd suggest to write:
```suggestion
SleepECG provides a consistent functional interface for downloading and reading common polysomnography datasets. While reader functions are a [WIP](https://github.com/cbrnr/sleepecg/pull/28), SleepECG already provides a Python interface for downloading datasets from the _National Sleep Research Resource_ (NSRR) on [sleepdata.org](https://sleepdata.org/), which replicates the functionality of the [NSRR Ruby Gem](https://github.com/nsrr/nsrr-gem).
```

Reply:
```suggestion
SleepECG provides a consistent functional interface for downloading and reading common polysomnography datasets. While reader functions are a [WIP](https://github.com/cbrnr/sleepecg/pull/28), SleepECG already provides an interface for downloading datasets from the _National Sleep Research Resource_ (NSRR) on [sleepdata.org](https://sleepdata.org/), which replicates the functionality of the [NSRR Ruby Gem](https://github.com/nsrr/nsrr-gem).
```

Reply:
OK like this (no need to explicitly mention Python)?

Reply:
I was hoping this might improve visibility if someone would search e.g. for "nsrr python", but I'm fine with omitting it as well!

Reply:
I see your point. In the text it sounds a bit awkward because SleepECG is a Python package, so of course the interface will be a Python interface. I think Google should/will be smart enough to associate Python with our NSRR mention, so I'll go ahead and merge that one.